Change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    fig, ax = plt.subplots(nrows=1, ncols=3)
</pre><p>These are subplots, so matplotlib default width is too less to show this figure in notebook (as you can observe). I suggest to adjust figsize conditionally, means</p><p>if LATEXIFY:</p><p>	figsize = None #because we do not want to interfere in latexify settings</p><p>else:</p><p>	figsize = (10,4)</p><p> </p><p>I have used this kind of thing in: https://github.com/karm-patel/pyprobml-1/blob/Fig2.6_datasaurus_dozen/notebooks/book1/datasaurus_dozen.ipynb</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>